values-schema.yaml

```diff
@@ -218,6 +218,40 @@ definitions:
         type: string
     title: Container image
     type: object
+  jobs:
```

Comment:
Could you define just a schema for a single job?

Reply:
Yes that makes sense.